ansible/ansible_collections/agentic/agent_configs/tests/unit/plugins/modules/test_agent_config_modules.py

```
@@ -0,0 +1,495 @@
+        "body": extra_args.get("body", "Run it."),
+        **extra_args,
+    }
+
+    result = run_module(module, args)
+
+    assert result["changed"] is True
+    assert result["path"] == str(path)
+    rendered = path.read_text()
+    for expected in expected_frontmatter:
+        assert expected in rendered
+    assert rendered.endswith((extra_args.get("body", "Run it.") + "\n"))
```

Ruff (0.15.11)

[warning] Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)